### PR TITLE
Add object deletion as a solution to nospace alarm recovery

### PR DESCRIPTION
**Pull request by geetasg**
@ahrtr @chaochn47 @cenkalti

### CODE REVIEW

File: content/en/blog/2023/how_to_debug_large_db_size_issue.md [hunk 1]
```diff
@@ -246,5 +246,7 @@ In the following example, there are two etcd clusters; one for the normal usage,
 --etcd-servers-overrides="/events#https://etcd-3:2379,https://etcd-4:2379,https://etcd-5:2379"
 ```
 
+Another solution would be to free up space taken up by unwanted objects.You will need to delete objects before running compaction and defragmentation to free up the space.
```
Comment (cenkalti) [inline content/en/blog/2023/how_to_debug_large_db_size_issue.md:249]: Need a space between sentences.

Comment (ahrtr) [reply content/en/blog/2023/how_to_debug_large_db_size_issue.md:249]:
```
As a mitigation, you may also want to delete unneeded objects to free up some space. You will 
need to delete objects before running compaction and defragmentation to free up the space.
```

**Comment by ahrtr** [comment]
please squash the commits.

**Review by jmhbnz — APPROVED** [review]
LGTM

**Review by ahrtr — APPROVED** [review]
LGTM

**Pull request merged**